buildSrc/build.gradle.kts

```diff
@@ -1,16 +1,6 @@
 // Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-val jacksonVersion: String by project
```

Comment:
They fixed this? Nice

Reply:
Sorttaaa? They are getting generated by the precomipled accessor system it seems.